lib/Fencer/Rules.hs

```diff
@@ -21,18 +27,36 @@ import qualified Data.Yaml as Yaml
 
 import Fencer.Types
 
+data LoadRulesError
+  = ParseException FilePath Yaml.ParseException
+  | IOEx IOException
+
+instance Show LoadRulesError where
+  show (ParseException file yamlEx) =
+    show file ++ ", " ++ (Yaml.prettyPrintParseException yamlEx)
+  show (IOEx ex) = "IO exception: " ++ show ex
+
+instance Eq LoadRulesError where
```

Comment:
Let's avoid an instance like that. If it's used in tests, you can just use ``(==) `on` show`` there.

Reply:
I implemented your suggestion in an upcoming commit.